TodayVideo/TodayVideo/Common/FilterButton.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FilterButton.swift
+//  TodayVideo
+//
+//  Created by iOS Dev on 1/16/25.
+//
+
+import UIKit
+
+class FilterButton: UIButton {
+    let height = 50.0
+    var title = ""
+    
+    init(title: String) {
+        super.init(frame: .zero)
+        
+        self.title = title
+        self.setTitle(title, for: .normal)
+        self.layer.cornerRadius = height / 2
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    func width() -> CGFloat {
+        let titleWidth = title.width(size: 20)
+        let leftRightMargin = 70.0
```

Comment:
```suggestion
        let leftRightMargin: CGFloat = 70.0
```

Reply:
70.0과 같은 매직 넘버는 상수로 바꾸어 가독성을 높일 수 있습니다. 예를 들어, `leftRightMargin` 값을 상수로 정의하고 재사용할 수 있습니다.

```suggestion
    private let leftRightMargin: CGFloat = 70.0
```